### ISSUE TITLE
Is there a timeline on when we can expect 0.10.0 to be released?

### ISSUE THREAD

**Issue by JOyo246** [nalexn/ViewInspector#299]
Just trying to understand the release schedule for VI. In particular, wanting inspect Toggle.
(Note: I know I could just target the release branch)

**Comment by nalexn** [comment]
It was unfortunate for that fix for toggle to be added just before we started an endeavor of adding structured concurrency support. It's mainly done, but I was hoping to get some more experimentation done before we release.
We actually can issue `0.9.11` with just that fix for toggle, would that make sense?

**Comment by JOyo246** [comment]
That would be awesome!

For the future, any thoughts on using feature branches for additions like structured concurrency?

**Comment by nalexn** [comment]
We had one, and it was merged when it was done, it's just later I discovered some nice to haves

**Comment by nalexn** [comment]
0.9.11 is out
